doc/installation/linux_install.rst

```diff
@@ -1,29 +1,50 @@
 Ubuntu/Debian Installation
-===============================
+==========================
 
 .. _standard:
 
-Installation from source
---------------------------
+Dependencies
+------------
+
+To build NEST, you need a recent version of `CMake <https://cmake.org/install>`_ and `libtool <https://www.gnu.org/software/libtool/libtool.html>`_; the latter should be available for most systems and is probably already installed.
```

Comment:
We should have a central file where dependencies are collected, as we have for the docs (conda environment). This section could then have an additional sentence "For the full list of requirements check [here]()". We should do this in a separate PR though.

Reply:
Thanks for adding #1612 for this, @sarakonradi.